### ISSUE TITLE
Support for none reference projects

### ISSUE THREAD

**Issue by cannehag** [frhagn/Typewriter#63]
Hi, we need support for generating ts files in a project without adding a reference to another project.
Example, in Project 2 we need to generate ts files for content in Project1 WITHOUT having a reference to Project1:
${
   import "Project1";
}

This import would simply include the Project1's file in the generation without needing a project reference.

**Comment by frhagn** [comment]
Hi @cannehag,
Great idea! I'm currently working on a settings API similar to your solution, and this could definetly be included there.

**Comment by cannehag** [comment]
Cool, let me know when you have something working then :)

**Comment by frhagn** [comment]
@cannehag, I just released Typewriter 1.2 which resolves this issue. You can now add a template constructor to configure settings om the template.
```
${
    Template(Settings settings)
    {
        settings
            .IncludeProject("Project1")
            .IncludeProject("Project2");
    }
}
```
